sdk/communication/Azure.Communication.PhoneNumbers/tests/samples/Sample2_PhoneNumberAdministrationClient.cs

```diff
@@ -467,6 +467,7 @@ public async Task GeographicalAreaCodesAsync()
 
         [Test]
         [SyncOnly]
+        [PlaybackOnly("TollFreeNumbersAreDisabledTemporarily")]
```

Comment:
Toll Free numbers are disabled, but also we are going to replace these tests completely for the new API anyways, so it is fine to change them to playback.